SQL/init_table.sql

```diff
@@ -52,3 +52,10 @@ CREATE TABLE tab_chat_message (
     FOREIGN KEY (id_user) REFERENCES tab_user (id) ON UPDATE NO ACTION ON DELETE NO ACTION,
     FOREIGN KEY (id_message) REFERENCES tab_message (id) ON UPDATE NO ACTION ON DELETE NO ACTION
 );
+CREATE TABLE tab_log (
+    id INTEGER NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    date_time DATETIME NOT NULL,
+    event_type VARCHAR(20) NOT NULL,
+    id_user INTEGER,
```

Comment:
```suggestion
    id_user INTEGER,
```
Не одобряю сию строку